address/get.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import { failure, success } from '../libs/response-lib';
+
+export const main = async event => {
+  const { postalCode, houseNumber } = event.pathParameters;
+  const addressEndpoint = process.env.AZURE_MAPS_ADDRESS_ENDPOINT;
+  const clientId = process.env.AZURE_MAPS_CLIENT_ID;
+  const mapsKey = process.env.AZURE_MAPS_KEY;
+
+  try {
+    const response = await axios.get(
+      `${addressEndpoint}?countryCode=NL&api-version=1.0`,
+      {
+        headers: {
+          'x-ms-client-id': clientId
+        },
+        params: {
+          'subscription-key': mapsKey,
+          postalCode: postalCode.replace(' ', ''),
+          houseNumber
+        }
+      }
+    );
+
+    const data = response.data.results[0];
```

Comment:
especially here it is impossible to know the fields of `data` unless one do the manual test

Reply:
I added some JSDoc, but not sure if it is the correct way. Never before have I used this approach honestly. If you see if this works. Webstorm doesn't provide me with any info whatsoever

Reply:
looks good, thanks!
VSCode supports it and does a clever suggestions